packages/flutter/lib/src/material/dialog.dart

```diff
@@ -41,16 +41,32 @@ class Dialog extends StatelessWidget {
   /// Typically used in conjunction with [showDialog].
   const Dialog({
     Key key,
-    this.child,
+    this.backgroundColor,
+    this.elevation,
     this.insetAnimationDuration = const Duration(milliseconds: 100),
     this.insetAnimationCurve = Curves.decelerate,
     this.shape,
+    this.child,
   }) : super(key: key);
 
-  /// The widget below this widget in the tree.
+  /// {@template flutter.material.dialog.backgroundColor}
+  /// The background color of the surface of this [Dialog].
   ///
-  /// {@macro flutter.widgets.child}
-  final Widget child;
+  /// This sets the [Material.color] on this [Dialog]'s [Material].
+  ///
+  /// If `null`, [ThemeData.cardColor] is used (this is the default for
+  /// [MaterialType.card]).
+  /// {@endtemplate}
+  final Color backgroundColor;
+
+  /// The z-coordinate of this [Dialog].
+  ///
+  /// If this is `null`, [DialogTheme.elevation] will be used, if
```

Comment:
If null then [DialogTheme.elevation] is used, and if that's null then the dialog's elevation is 24.0.

Reply:
Done.